java/src/IceGridGUI/src/main/java/com/zeroc/IceGridGUI/Application/PlainServer.java

```diff
@@ -73,7 +68,7 @@ static public ServerDescriptor newServerDescriptor()
             "",
             "",
             true,
-            new DistributionDescriptor("", new java.util.LinkedList<String>()),
+            new DistributionDescriptor("", new java.util.LinkedList<String>()), // dummy
```

Comment:
We have to keep this for backward compatibility, right?

Reply:
Yes. It's not so much for the IceGridGUI - IceGrid communications but for the database format.